libs/components/forms/src/lib/modules/file-attachment/file-attachment.component.spec.ts

```diff
@@ -1597,6 +1597,23 @@ describe('File attachment', () => {
     ).toBe(1);
   });
 
+  it('should not render help inline if help key is set', () => {
+    fixture.componentInstance.showLabel = false;
+    fixture.componentInstance.labelText = 'labelText';
+    fixture.detectChanges();
+
+    expect(
+      fixture.nativeElement.querySelectorAll('sky-help-inline').length,
```

Comment:
Instead of checking length here, I'd use `querySelector` and look for `null` or a falsy value:

```
expect(fixture.nativeElement.querySelector('sky-help-inline').toBeFalsy();
expect(fixture.nativeElement.querySelector('sky-help-inline').toBeTruthy();
```